packages/react-components/react-spinbutton/src/stories/SpinButton.stories.tsx

```diff
@@ -43,12 +27,8 @@ const meta: Meta = {
   },
   decorators: [
     (Story, context) => {
-      const decoratorStyles = useDecoratorStyles();
-
-      const className = mergeClasses(decoratorStyles.base, context.viewMode === 'docs' && decoratorStyles.docsViewMode);
-
       return (
-        <div className={className}>
+        <div>
           <Story />
         </div>
```

Comment:
Is this needed? You replaced the fragment with a div in the stories and this dive has no styling.

Reply:
Good call! Removed.